### PR TITLE
condafi diffbind, update requirements

### PR DESCRIPTION
**Pull request by martenson**
Diffbind wrapper has no tests, but it is broken anyway, xref. galaxyproject/galaxy#2318 .

Alternative to bgruening#423 .

### CODE REVIEW

File: tools/diffbind/diffbind.xml [hunk 1]
```diff
@@ -1,11 +1,9 @@
-<tool id="diffbind" name="DiffBind" version="2.1.0">
+<tool id="diffbind" name="DiffBind" version="2.1.16.3">
     <description> differential binding analysis of ChIP-Seq peak data</description>
     <requirements>
-        <requirement type="binary">Rscript</requirement>
-        <requirement type="R-module">DESeq2</requirement>
-        <requirement type="package" version="3.0.3">R</requirement>
-        <requirement type="package" version="1.2.10">deseq2</requirement>
-        <requirement type="package" version="1.8.3">diffbind</requirement>
+        <requirement type="package" version="3.3.2">r</requirement>
```
Comment (lecorguille) [inline tools/diffbind/diffbind.xml]: You don't really need R. Conda will do the job

File: tools/diffbind/diffbind.xml [hunk 2]
```diff
@@ -1,11 +1,9 @@
...
+        <requirement type="package" version="1.12.4">deseq2</requirement>
```
Comment (lecorguille) [inline tools/diffbind/diffbind.xml]: The good name is bioconductor-deseq2

File: tools/diffbind/diffbind.xml [hunk 3]
```diff
@@ -1,11 +1,9 @@
...
+        <requirement type="package" version="1.16.3">diffbind</requirement>
```
Comment (lecorguille) [inline tools/diffbind/diffbind.xml]: The same the package name is bioconductor-diffbind

File: tools/diffbind/diffbind.xml (hunk 2 above)
Comment (lecorguille) [inline tools/diffbind/diffbind.xml]: Deseq2 seems already include within the bioconductor-diffbind onda package. So you can omit it :)

**Comment by martenson** [comment]
superseded by #524

thanks for the review @lecorguille !

**Comment by lecorguille** [comment]
No problemo :)

**Pull request closed without merging**